Cargo.toml

```
@@ -20,12 +20,12 @@ default = []
 enable = ["svgbob"]
 
 [dependencies]
-syn = "1.0.41"
+syn = "2.0"
 quote = "1"
-svgbob = { version = "= 0.6.6", optional = true }
+svgbob = { version = "0.7", optional = true }
 proc-macro2 = "1"
-base64 = ">= 0.5.2, < 0.14"
-unicode-width = "0.1"
+base64 = "0.22"
+unicode-width = "0.2"
```

The difference between `^0.1` and `^0.2` is insignificant for us (afaik), but `svgbob ^0.7` still uses `^0.1`, so forcing the bump to `^0.2` would increase the build time.

```suggestion
unicode-width = ">= 0.1, < 0.3"
```